runtime/src/bank/tests.rs

```diff
@@ -2821,7 +2817,6 @@ fn test_bank_blockhash_fee_structure() {
     assert_eq!(bank.get_balance(&key), 1);
     let cheap_fee = calculate_test_fee(
         &new_sanitized_message(Message::new(&[], Some(&Pubkey::new_unique()))),
-        cheap_lamports_per_signature,
```

Comment:
Isn't this test kind of moot now? Sure, it's checking that the blockhash LPS is changing according to the governor, but that isn't actually influencing fees at all. I guess it was moot to begin with?